aws-android-sdk-ddb-mapper/src/main/java/com/amazonaws/mobileconnectors/dynamodbv2/dynamodbmapper/DynamoDBQueryExpression.java

```diff
@@ -352,6 +353,37 @@ public void setIndexName(String indexName) {
         return this;
     }
 
+    /**
+     * Returns the key condition expression to be used by this query.
+     *
```

Comment:
If possible, can you add an example of one of them here, and to the `with.*()` method?

Reply:
linked to DDB guide doc